game/management/commands/runserver.py

```
@@ -0,0 +1,29 @@
+    help = (
+        'Starts a lightweight web server for development and reminds '
+        'contributors to open the site over HTTP.'
+    )
+
+    def on_bind(self, server_port):
+        super().on_bind(server_port)
+
+        if self._raw_ipv6:
+            host = f'[{self.addr}]'
+        elif self.addr == '0':
+            host = '0.0.0.0'
```

Ruff (0.15.14)

[error] Possible binding to all interfaces

(S104)